doc/en/reference.rst

```diff
@@ -1039,6 +1039,10 @@ passed multiple times. The expected format is ``name=value``. For example::
 
    pytest -o console_output_style=classic -o cache_dir=/tmp/mycache
 
+Multi-valued configuration options may be appended to in the command-line by using ``-a/-append-ini``, which can also be passed multiple times.
+The expected format is ``name=value``. For example::
+
+   pytest --append-ini addopts="-k should_throw_error" -a addopts=-x
```

Comment:
I think we should tweak this example because

1. According to current code in the PR at least, `--append-ini` on its own (without a base value) is illegal
2. Using `addopts` for an example of `--append-ini` is a little funny because it's equivalent to just passing the args directly

Reply:
Perhaps a good example would be passing `filterwarnings`, as that is also a list.

Ditto for the example in the docstring. 👍